renderapi/image.py

```diff
@@ -33,14 +33,26 @@ def get_bb_image(stack, z, x, y, width, height, scale=1.0,
                  render=None, **kwargs):
     '''
     render image from a bounding box defined in xy and return numpy array:
-        z: layer
-        x: leftmost point of bounding rectangle
-        y: topmost pont of bounding rectangle
-        width: extent to right in x
-        height: extent down in y
-        binaryMask: optional, boolean whether to treat maskimage as binary
-        maxTileSpecsToRender: optional, int number of tilespecs to render
-        filter: optional, boolean whether to use Khaled's preferred filter
+
+    :func:`renderapi.render.renderaccess` decorated function
+
+    Args:
+        stack (str): name of render stack to get image from
+        z (float): z value to render
+        x (int): leftmost point of bounding rectangle
+        y (int): topmost pont of bounding rectangle
+        width (int): number of units @scale=1.0 to right (+x() of bounding box to render
+        height (int): number of units @scale=1.0 down (+y) of bounding box to render
+        scale (float): scale to render image at (default 1.0)
+        binaryMask (boolean): whether to treat maskimage as binary
+        maxTileSpecsToRender (int):  max number of tilespecs to render
+        filter (boolean): whether to use server side filtering
+        render (renderapi.render.Render): render connect object
+        session (requests.sessions.Session): sessions object to connect with
+    Returns:
+        numpy.array: [N,M,3] array of image data from render

```

Comment:
The shape of this numpy array depends on the function inputs.  e.g., png-image will return [N,M,4] as it includes an alpha channel.  I think [N, M, :] may be better as this is very format dependent (and you could imagine parameters changing this as well).